apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -233,18 +233,49 @@ public Response getOffers(  @ApiParam(value = "Type of the offer. (BUY = 0 /SELL
         ).build();
     }
 
-    @GET
-    @Path("/order")
+    @POST
+    @Path("/offer/cancel")
```

Comment:
PUT should be idempotent, in our case we will have an Exception call twice. 
DELETE is also doesn't fit here, because we actually don't delete anything.